src/lib/storage/s3.ts

```
@@ -0,0 +1,38 @@
+
+export function isS3Enabled() {
+  return Boolean(process.env.AWS_S3_BUCKET && process.env.AWS_ACCESS_KEY_ID && process.env.AWS_SECRET_ACCESS_KEY && process.env.AWS_S3_REGION)
+}
+
+export function getS3(): any | null {
+  if (!isS3Enabled()) return null
+  if (client === undefined) {
+    try {
+      // Lazy import to avoid dependency requirement when not configured
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      awsSdk = require('@aws-sdk/client-s3')
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)